test/cpp_interface_test.cpp

```diff
@@ -24,6 +24,15 @@ namespace {
 using ::testing::ContainerEq;
 using ::testing::HasSubstr;
 
+// Return a string that contains the minimum instructions needed to form
+// a valid module.  Other instructions can be appended to this string.
+std::string Header() {
```

Comment:
Does this need to be a method? What about just having a

const char[] kHeader = R"( ... )";

Reply:
Yes you are right, that is a POD, so I can be a global.  However, I kinda want an std::string to make concatenation simpler.  Since the string is not POD, I cannot have a global std::string.

Reply:
Doesn't concat just work? I thought it would just turn the c-str into a std::string as needed?